internal/adapters/consul/http.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/hashicorp/consul/api"
 )
 
-// httpRouteToServiceDiscoChain will convert a k8s HTTPRoute to a Consul service-router config entry and 0 or
+// httpRouteDiscoveryChain will convert a k8s HTTPRoute to a Consul service-router config entry and 0 or
```

Comment:
👍 I believe this was originally written as `DiscoChain` since it's a well-known internal nickname for discovery chains in core, but I much prefer the explicit name, and that way it doesn't make me think about these guys:

![Bee Gees](https://media0.giphy.com/media/xUPGctGBglZmaSH9cI/giphy.gif)